modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disable-individual-bootsource-update_{context}"]
+= Disabling an automatic update for a custom boot source
+
+You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+
+.Procedure
+
+* Edit the HyperConverged custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      annotations:
+        dataimportcrontemplate.kubevirt.io/enable: false
+      name: rhel8-image-cron
```

Comment:
Add `...` after this line to indicate that the file continues

Reply:
Done.